### ISSUE TITLE
How to add custom currency

### ISSUE THREAD

**Issue by dejurin** [python-babel/babel#790]
For example, I need added BTC. Is there any way?

**Comment by FlorBl** [comment]
Yes, you can get up-to-date BTC (other crypto-currencies as well) by using an API. There's a couple of free APIs you can use. You just have to register and follow the instructions.
